packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -4797,6 +4797,115 @@ QUnit.module('the \'acceptCustomValue\' option', moduleSetup, () => {
     });
 });
 
+QUnit.module('the \'deferRendering\' option', () => {
+    [true, false].forEach(deferRendering => {
+        QUnit.test(`if ${deferRendering} should not display any errors [T1230995]`, function(assert) {
+            try {
+                const $element = $('#tagBox').dxTagBox({
+                    applyValueMode: 'useButtons',
+                    acceptCustomValue: true,
+                    onCustomItemCreating(args) {
+                        const newValue = args.text;
+                        const { component } = args;
+                        const currentItems = component.option('items');
+                        const isItemInDataSource = currentItems.some((item) => item === newValue);
+                        if(!isItemInDataSource) {
+                            currentItems.unshift(newValue);
+                            component.option('items', currentItems);
+                        }
+                        args.customItem = newValue;
+                    },
+                    deferRendering
+                });
+                const $input = $element.find(`.${TEXTEDITOR_INPUT_CLASS}`);
+                const keyboard = keyboardMock($input, true);
+                keyboard
+                    .focus()
+                    .type('test 1')
+                    .press('enter');
+
+                assert.ok(true, 'Error not encountered');
+            } catch(error) {
+                assert.ok(false, `Error encountered is: ${error.message}`);
+            }
+        });
+
+        QUnit.test(`if ${deferRendering} should display right selected items [T1230995]`, function(assert) {
```

Comment:
The caption is nor informative: `if true should....`

Reply:
[30d85eb](https://github.com/DevExpress/DevExtreme/pull/27341/commits/30d85eb8b885fb8cfad6063c4b073a2970efa847)